model/k8s/registry.yaml

```diff
@@ -57,18 +57,14 @@ groups:
         type: template[string]
         stability: development
         brief: >
-          The label key-value pairs placed on the Node.
-        note: |
-          The `<key>` being the label name, the value being the label value, even if the value is empty.
-        examples: ['k8s.node.label.kubernetes.io/arch=arm64', 'k8s.node.label.data=']
+          The label key-value pairs placed on the Node, the `<key>` being the label name, the value being the label value, even if the value is empty.
+        examples: ['arm64', '']
```

Comment:
I find this format less useful/explanatory compared to the full one `'k8s.node.label.kubernetes.io/arch=arm64'`.  Is there a plan to have the labels keys still in place in the future or they are going to be removed completely? It has always been useful to have the full representation for the k8s labels.

Reply:
Fair enough. Do you think something like this would be a better long-term story - https://github.com/open-telemetry/semantic-conventions/issues/1740#issuecomment-2833598168? 

```yaml
- id: http.request.header
  type: template[string[]]
  brief: The value of the specific HTTP request header.
    key: The normalized (lowercase) HTTP header name.
    value: List of all values with this header name provided in the request.
  examples:
    key: ["content-type", "x-forwarded-for"]
    value: [["application/json"], ["1.2.3.4", "1.2.3.5"]]
```

Effectively we'd formalize what we do in yaml inconsistently today.

Reply:
That would be better yes, specially if in the produced markdowns we combine these together to produce the human readable `'k8s.node.label.kubernetes.io/arch=arm64'`.

Reply:
My suggestion was subtly different but kept the pairing:
```yaml
- id: http.request.header
  type: template[string[]]
  brief: The value of the specific HTTP request header.
    key: The normalized (lowercase) HTTP header name.
    value: List of all values with this header name provided in the request.
  examples:
    - key: "content-type"
      value: ["application/json"]
    - key: "x-forwarded-for"
      value: ["1.2.3.4", "1.2.3.5"]
```

Reply:
Thank you, looks good! From my side feel free to resolve this thread.